txscript/reference_test.go

```diff
@@ -158,6 +158,8 @@ func parseScriptFlags(flagStr string) (ScriptFlags, error) {
 			flags |= ScriptVerifyMinimalData
 		case "NONE":
 			// Nothing.
+		case "NULLFAIL":
```

Comment:
I think this should be after `NULLDUMMY` so it maintains the alphabetical order the rest have.